### PR TITLE
feat: show empty search results #2527

### LINKED ISSUE #2527: Show empty search results

### PR DESCRIPTION
**Pull request by frolyo**

### CODE REVIEW

File: src/dotnet/UI.Blazor.App/Components/NoResults/NoResults.razor [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@namespace ActualChat.UI.Blazor.App.Components
```
Comment (crui3er) [inline src/dotnet/UI.Blazor.App/Components/NoResults/NoResults.razor:1]: Why did you put here? Not next to FoundChatList?

File: src/dotnet/UI.Blazor.App/Components/NoResults/no-results-svg.lit.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { customElement } from 'lit/decorators.js';
+import { html, LitElement } from 'lit';
+
+@customElement('no-results-svg')
```
Comment (crui3er) [inline src/dotnet/UI.Blazor.App/Components/NoResults/no-results-svg.lit.ts:4]: Why is this svg wrapped into a lit component?

Comment (frolyo) [reply src/dotnet/UI.Blazor.App/Components/NoResults/no-results-svg.lit.ts:4]: the same way as the other not found views

File: src/dotnet/UI.Blazor.App/Components/NoResults/NoResults.razor (hunk 1 above)
Comment (frolyo) [reply src/dotnet/UI.Blazor.App/Components/NoResults/NoResults.razor:1]: It's just a special cat component, so I decided to put it along with other cat components

**Pull request merged**